app.js

```diff
@@ -174,6 +175,8 @@ app.get('/sessions/current', async function (req, res, next) {
       return res.header('mu-auth-allowed-groups', 'CLEAR').status(403).end();
     }
 
+    await insertLoginActivity(session.userUri);
```

Comment:
Don't we want to do this in the `POST /sessions` endpoint as well to capture logins? The `/sessions/current` endpoint isn't called then.

Reply:
For the `POST /sessions` call it happens in the `ensureUserResources` call, because we have the personUri there. But it does look a bit funky with it happening on separate levels, so I can make the `ensureUserResources` call return the needed data and insert the login activity in the top file.